requirements.txt

```
@@ -1,2 +1,2 @@
-data-horizon[client-sync]>=0.0.8,<0.1
+data-horizon[client-sync]>=0.0.8
```

```suggestion
data-horizon[client-sync]>=0.0.8,<0.2
```

Не надо совсем удалять верхний лимит, пусть будет, для предотвращения потенциальных проблем в будущем